sorry.html

```diff
@@ -24,19 +24,27 @@
             padding: 30px;
         }
     </style>
+
+    <script>
+        // Firefox in some modes, on some systems, seems to end up with an extremely tall `body` that makes
+        // it impossible to see the page content, without this workaround. For our current purposes we expect
+        // it to be harmless to run on all browsers.
+        setTimeout(function() {
+          document.body.style.paddingTop = 0;
+          document.body.style.maxHeight = '200%';
+        }, 1_000);
```

Comment:
Why does this function need to be delayed with `setTimeout`? And is the specific time of 1s important?

Reply:
I don't have great answers on this, but _no_ delay (in `window.onload`) seems to be insufficient to let Firefox correct the weird rendering.

0.5s was OK on my local but if there's a dependency between this and loading the CDN files fully then that seems aggressive. And 1s seemed easily fast enough that visitors are unlikely to give up in most cases.

Reply:
Added more or less this to the code.

Reply:
Right. 1s seems like it should be fine. If it has to be waiting until the stuff it needs to correct is run then it might be more robust to use `setInterval` instead and just run this function repeatedly, like every 300ms or something.